src/_includes/components/landing/dev.njk

```diff
@@ -1,12 +1,12 @@
 <section id="dev">
-  <h1>Package a <em>service</em></h1>
+  <h1><em>Package</em> a service</h1>
   <div class="cols">
     <div class="col">
       <div class="wrapper">
         <div class="content">
-          <p>Are you a developer? Have you created open source software? Do you want a distribution platform for your project?</p>
-          <p>Package a service for embassyOS and get the word out.</p>
-          <p>Help lead the movement. <em>Our platform runs on devs like you.</em></p> 
+          <p>Packaging a service for emabssyOS is a <em>challenging, exciting, creative, and rewarding</em> experience.</p>
+          <p>If it can be self-hosted, it can be run on embassyOS.</p>
```

Comment:
```suggestion
          <p>If it can be self-hosted, it can run on embassyOS.</p>
```

Reply:
i like the consistency of "be" in the two statements, but "be run" sounds kind of weird?